epub33/rs/index.html

```diff
@@ -299,6 +299,13 @@ <h4>Data URLs</h4>
 					content documents.</p>
 			</section>
 
+			<section id="sec-epub-rs-conf-file-urls">
+				<h4>File URLs</h4>
+				<p id="confreq-rs-file-urls">
+					Reading systems MUST prevent file URLs [[rfc8089]] from opening.
+				</p>
```

Comment:
I wonder if "opening" is the right term. That seems to cover a scenario like trying to load a file reference into an iframe, but is a script that uses fetch or xhr to get the data of the file "opening" the file?

What about something like: "Reading Systems MUST prevent access to resources referenced from file URLs [[rfc8089]], both from elements that embed resources and from script API calls." ?

Reply:
The term "embed" does not sound right to my ears, it seems to say that something like `<a href="file:...">` is allowed. 

I have used:

> Reading Systems MUST prevent access to resources referenced via file URLs [[rfc8089]], both from elements that refer to resources and from script API calls.

Reply:
I don't like using both here, since it implies those are the only two cases. I would drop the list, which seems fine. If we think we need to clarify when and how file URLs are used, we should list this as potential examples. But it feels like "Reading Systems MUST prevent access to resources referenced via file URLs [[rfc8089]]" full stop is exactly enough, and tacking on more just introduces more surfaces for confusion.

Reply:
That's fine with me, or even just putting the examples in a parenthetical "e.g." to make clear its not a complete list.

It was more a reaction to the limits that "opening" resources has, but is not strictly necessary.

Reply:
Nuked the list :-)